src/es_sfgtools/processing/operations/gnss_resources.py

```diff
@@ -24,18 +26,261 @@ def _parse_date(date:datetime.date | datetime.datetime)-> Tuple[str,str]:
     elif doy < 100:
         doy = f"0{doy}"
```

Comment:
what if doy<10?  wouldn't you need 2 leading zeros?

Reply:
that is handled in line 24